myuw/templates/handlebars/banner/profile.html

```diff
@@ -37,24 +37,33 @@ <h3 class="sr-only">Official name and student ID number</h3>
 
             {{#with password}}
             <div class="profile-passwords row">
-                <h3 class="sr-only">Password Information</h3>
-                {{#if has_active_med_pw}}
-                <div class="col-md-6 uwmedicine-password">
-                    <h4>UW Medicine Account</h4>
-                    {{#if med_pw_expired }}
-                        <span class="password-alert">Password expired on {{formatDateAsDate expires_med}}. <a href="https://services.uwmedicine.org/passwordportal/login.htm">Change your password to regain access.</a></span>
```

Comment:
@wmwash Not sure if you want this, but you could add `target="_blank"` to open these links in a new window.